server/graphql/v1/CollectiveInterface.js

```diff
@@ -1513,12 +1517,25 @@ const CollectiveFields = () => {
           defaultValue: false,
           description: 'Defines if the organization "collective" payment method should be returned',
         },
+        maxExpiryDate: {
+          type: GraphQLString,
+          description: 'Find Payment Methods that have expired before a particular date',
+        },
       },
       async resolve(collective, args, req) {
         if (!req.remoteUser || !req.remoteUser.isAdmin(collective.id)) {
           return [];
         }
-        let paymentMethods = await req.loaders.PaymentMethod.findByCollectiveId.load(collective.id);
+        let paymentMethods = [];
+        // Load Payment Methods that have expired before a particular date
+        if (args.maxExpiryDate && new Date(args.maxExpiryDate) !== 'Invalid Date') {
+          paymentMethods = await req.loaders.PaymentMethod.findByCollectiveIdWithMaxExpiryDate.load({
+            collectiveId: collective.id,
+            maxExpiryDate: args.maxExpiryDate,
+          });
```

Comment:
We don't want to maintain two loaders for this. Let's include the gift cards expired for less than a month in the loader and do the filtering in JS.